Heejung/Lv2/오픈채팅방.js

```diff
@@ -0,0 +1,17 @@
+function solution(record) {
+  let answer = [];
+  let user = {}; // { id: nickname }
+  let recordArr = record.map((v) => v.split(' '));
+
+  // Enter, Change일 때 user 셋팅
+  for (let x of recordArr) {
+    if (x[0] !== 'Leave') user[x[1]] = x[2];
+  }
+
+  for (let x of recordArr) {
+    if (x[0] === 'Enter') answer.push(user[x[1]] + '님이 들어왔습니다.');
+    if (x[0] === 'Leave') answer.push(user[x[1]] + '님이 나갔습니다.');
+  }

```

Comment:
```suggestion
  recordArr.forEach((x) => {
    if (x[0] === 'Enter') answer.push(`${user[x[1]]}님이 들어왔습니다.`);
    if (x[0] === 'Leave') answer.push(`${user[x[1]]}님이 나갔습니다.`);
  })
```
문자열을 +로 이어붙이는 방법도 있지만 백틱을 사용해서 작성하는 방법도 있으니 상황에 따라 선택해서 사용하시면 좋을거 같고, 저도 잘 못하지만... 콜백함수의 매개변수명도 의미있게 지어주시는게 좋아요.